packages/bot-skeleton/src/scratch/hooks/field.js

```diff
@@ -49,7 +49,7 @@ const FieldCheckbox = () => {
                 Blockly.utils.createSvgElement(
                     'path',
                     {
-                        fill: '#FFF',
+                        fill: getComputedStyle(document.documentElement).getPropertyValue('--text-general'),
```

Comment:
I don't believe we need to use `getComputedStyle` here, it's an expensive call and I believe you can just pass the CSS variable and it should render the correct colour. The attributes specified in the second argument of `Blockly.utils.createSvgElement` are set in the DOM.